lib/tree.rb

```diff
@@ -1,25 +1,41 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+class Tree
+  attr_reader :height, :age
+  MAXIMUM_AGE = 100
```

Comment:
FYI: You'll sometimes see `.freeze` used in cases where magic strings & ints are defined.
http://ruby-doc.org/core-2.2.3/Object.html#method-i-freeze